ext/standard/tests/url/parse_url_basic_011.phpt

```diff
@@ -1,30 +0,0 @@
---TEST--
-Test parse_url() function: can not recognize port without scheme
---FILE--
-<?php
-echo "*** Testing parse_url() :can not recognize port without scheme ***\n";
```

Comment:
Maybe keep the test and change the expectation.

Reply:
Good point, added